src/System.Net.Requests/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netstandard;
+      uap-Windows_NT;
```

Comment:
#20136 is our epic work item for System.Net.Requests for UWP.

@tarekgh Is there a different issue number tracking the work fixing RemoteExecutor for UAP runs?  If so, I can use that issue number for the active issue for these tests disables.